### ISSUE TITLE
Validation Data Problem

### ISSUE THREAD

**Issue by ghost** [edgarschnfld/CADA-VAE-PyTorch#10]
I don't see you split the trainval data into train and validation, so how do you select model parameters without validation data?

**Comment by edgarschnfld** [comment]
Hi, the model parameters where chosen on the validation split that was proposed by Xian et al (https://arxiv.org/abs/1707.00600). I will add an option to select the validation splits for training and push it to this repository later this evening.

**Comment by yaoweihu** [comment]
thanks. I download the datasets proposed by Xian, but in the file "att-splits", I just see "trainval_loc" no "train_loc" or "val_loc". Meanwhile, there is also no mapping between classname and label number, so I don't know how to split the trainval_loc into train_loc and val_loc.

**Comment by edgarschnfld** [comment]
I know, it's confusing. I created the branch "validation". Everything there is the same, except data_loader.py. The dataloader class defined in there has an extra argument "validation_mode" which you can set to True. I wrote that a while back but it should work. (When I cleaned the code for this repo I left out the validation-split code, and had to rewrite it later). If everything is fine I'll merge it with the master branch tomorrow.
